src/wayland/symbols.map

```diff
@@ -445,6 +445,20 @@ global:
     vtable?for?mir::wayland::InputMethodV2;
     virtual?thunk?to?mir::wayland::InputMethodV2::?InputMethodV2*;
 
+    mir::wayland::IdleInhibitManagerV1::*;
+    non-virtual?thunk?to?mir::wayland::IdleInhibitManagerV1::*;
+    typeinfo?for?mir::wayland::IdleInhibitManagerV1;
+    vtable?for?mir::wayland::IdleInhibitManagerV1;
+    typeinfo?for?mir::wayland::IdleInhibitManagerV1::Global;
+    vtable?for?mir::wayland::IdleInhibitManagerV1::Global;
+    virtual?thunk?to?mir::wayland::IdleInhibitManagerV1::?IdleInhibitManagerV1*;
+
+    mir::wayland::IdleInhibitorV1::*;
+    non-virtual?thunk?to?mir::wayland::IdleInhibitorV1::*;
+    typeinfo?for?mir::wayland::IdleInhibitorV1;
+    vtable?for?mir::wayland::IdleInhibitorV1;
+    virtual?thunk?to?mir::wayland::IdleInhibitorV1::?IdleInhibitorV1*;
```

Comment:
These do not belong in the MIRWAYLAND_2.5 stanza - they were not introduced in Mir 2.5.

Reply:
Are you saying that changes to `symbols.map` are no longer needed?

Reply:
No. That, as they will be introduced in Mir 2.8, they should go in a 2.8 stanza.